src/builder.ts

```diff
@@ -36,6 +38,7 @@ const builder = (defaults: Partial<OrkaOptions> = _defaults) => {
   const middlewares: Middleware<any>[] = [];
 
   const _ = {
+    server: null as Server,
```

Comment:
Do you want to expose that to the clients? I would suggest moving this to L38 instead.

Reply:
Done.